### PR TITLE
[Issue #318] Added a script for ingesting population datasets

### PR DESCRIPTION
**Pull request by luochen01**
Added a script `ingestTwitterToLocalCluster.sh` for ingesting population datasets, which are similar to `ingestTwitterToLocalCluster.sh`.

Also added a script `ingestAllTwitterToLocalCluster.sh` to ingest all TwitterMap related data (as a wrapper).

### CODE REVIEW

File: script/ingestPopulationToLocalCluster.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+# ddl to register the twitter dataset
+cat <<'EOF' | curl -XPOST --data-binary @- http://localhost:19002/aql
+use dataverse twitter;
+create type typeStatePopulation if not exists as open{
+    name:string,
+    population:int64,
+    stateID:int64
+}
+create dataset dsStatePopulation(typeStatePopulation) if not exists primary key stateID;
+create index stateIDIndex if not exists on dsStatePopulation(stateID) type btree;
```
Comment (JavierJia) [inline script/ingestPopulationToLocalCluster.sh]: this index won't be required I think?

File: script/ingestPopulationToLocalCluster.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+echo 'Ingested county population dataset.'
+
+cat ./noah/src/main/resources/population/adm/allCityPopulation.adm | ./script/fileFeed.sh 10004
+echo 'Ingested city population dataset.'
+
+cat <<'EOF' | curl -XPOST --data-binary @- http://localhost:19002/aql
+use dataverse twitter;
+stop feed CityPopulationFeed;
+stop feed CountyPopulationFeed;
+stop feed StatePopulationFeed;
```
Comment (JavierJia) [inline script/ingestPopulationToLocalCluster.sh]: maybe also drop those feed as well?

File: script/ingestPopulationToLocalCluster.sh (hunk 1 above)
Comment (luochen01) [inline script/ingestPopulationToLocalCluster.sh]: Oh yes, it's duplicated. Surprisingly AsterixDB does not complain about it...

File: script/ingestPopulationToLocalCluster.sh (hunk 2 above)
Comment (luochen01) [inline script/ingestPopulationToLocalCluster.sh]: Sure, added drop feed statements.

File: script/ingestPopulationToLocalCluster.sh [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+create type typeCountyPopulation if not exists as open{
+    name:string,
+    population:int64,
+    countyID:int64,
+    stateName:string,
+    stateID:int64
+}
+create dataset dsCountyPopulation(typeCountyPopulation) if not exists primary key countyID;
+create index countyIDIndex if not exists on dsCountyPopulation(countyID) type btree;
```
Comment (JavierJia) [inline script/ingestPopulationToLocalCluster.sh]: same here?

File: script/ingestPopulationToLocalCluster.sh [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    name:string,
+    population:int64,
+    cityID:int64,
+    countyName:string,
+    countyID:int64,
+    stateName:string,
+    stateID:int64
+}
+create dataset dsCityPopulation(typeCityPopulation) if not exists primary key cityID;
+create index cityIDIndex if not exists on dsCityPopulation(cityID) type btree;
```
Comment (JavierJia) [inline script/ingestPopulationToLocalCluster.sh]: and here?

Comment (luochen01) [inline script/ingestPopulationToLocalCluster.sh]: Fixed.

File: script/ingestPopulationToLocalCluster.sh (hunk 3 above)
Comment (luochen01) [inline script/ingestPopulationToLocalCluster.sh]: Fixed.

**Pull request merged**